src/main/java/mate/academy/dao/OrderDao.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.dao;
+
+import java.util.List;
+import mate.academy.model.Order;
+import mate.academy.model.User;
+
+public interface OrderDao {
+
+    Order add(Order order);
+
+    List<Order> getOrdersHistory(User user);
```

Comment:
According to README.MD:

- To get the order for a specific user create the method getByUser().